<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true" extracomment="这是学生管理系统">信院学生信息管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/user2.jpg</normaloff>:/user2.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_simulation">
      <property name="text">
       <string>测试数据</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>175</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="3">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>学生信息管理系统</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>增加</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">image: url(:/user2.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="9">
     <widget class="QTableWidget" name="tableWidget">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>序号</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
         <kerning>true</kerning>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>年龄</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>年级</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>班级</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学号</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>电话</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>微信</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名:admin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="checkBox">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>全选</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6" colspan="3">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="9">
     <widget class="QPushButton" name="pushButton_6">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>清空学生表</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
